trestle/core/commands/author/component.py

```diff
@@ -212,7 +207,7 @@ def assemble_component(
             trestle_root, assem_comp_name, comp.ComponentDefinition, new_content_type
         )
 
-        if assem_comp_path.exists():
+        if not version and assem_comp_path.exists():
```

Comment:
Are both of those cases covered by tests? i.e. version is specified or not

Reply:
Yes - all those lines are covered by running separately with and without version specified.